addons/mail/static/src/components/thread_view/thread_view.scss

```diff
@@ -57,6 +62,10 @@
     background-color: gray('100');
 }
 
+.o_ThreadView_composer.o-has-current-partner-avatar {
+    background-color: $white;
+}
+
```

Comment:
This style should be in the SCSS of the `Composer` component. Awareness of `Composer` being in a `ThreadView` should not come from the SCSS style rule `o_ThreadView_composer` but from relation in models `this.composerView.threadView` in the `Composer` component.

If need be, should introduce classname modifiers in the template of the `Composer` component, so that we can target style rules specifically for that in SCSS.